presto-oracle/src/main/java/io/prestosql/plugin/oracle/OracleClientModule.java

```diff
@@ -43,6 +45,7 @@ public void configure(Binder binder)
         binder.bind(JdbcClient.class).annotatedWith(ForBaseJdbc.class).to(OracleClient.class).in(Scopes.SINGLETON);
         bindSessionPropertiesProvider(binder, OracleSessionProperties.class);
         configBinder(binder).bindConfig(OracleConfig.class);
+        configBinder(binder).bindConfigDefaults(JdbcMetadataConfig.class, config -> config.setDomainSizeThreshold(ORACLE_MAX_LIST_EXPRESSIONS));
```

Comment:
This is reasonable default for Oracle, but it is also a hard max limit -- for both config and session property. 
`fullPushdownIfSupported` was preventing queries from failing, now it becomes user's responsibility.